templates/keras_cv/index.md

```diff
@@ -0,0 +1,48 @@
+# KerasCV
+
+---
+## Quick links
+
+* [Getting started with KerasTuner](/guides/keras_tuner/getting_started/)
```

Comment:
Remember to update link locations from keras_tuner -> keras_cv here.

Reply:
Great catch - thanks.